bin/rialto-runtime/Cargo.toml

```diff
@@ -150,6 +155,12 @@ tag = 'v2.0.0-rc6'
 default-features = false
 git = "https://github.com/paritytech/substrate/"
 
+[dependencies.sp-finality-grandpa]
```

Comment:
Are you sure it is required?

Reply:
The stuff remaining in the Rialto runtime crate were missed during my initial runtime cleanup, thanks for catching it